library/tasks/CreateCountMatrix.wdl

```diff
@@ -3,6 +3,27 @@ task DropSeqToolsDigitalExpression {
   File whitelist
   Int estimated_required_disk = ceil((size(bam_input, "G") + size(whitelist, "G")) * 4.0) + 1
 
+  # runtime values
+  String docker = "quay.io/humancellatlas/secondary-analysis-dropseqtools:v0.2.2-1.12"
+  Int machine_mem_mb = 7500
+  Int cpu = 1
+  Int disk = ceil((size(bam_input, "G") + size(whitelist, "G")) * 4.0) + 1
+  Int preemptible = 0
+
+  meta {
+    description: "AMBROSE HALP!!"
+  }
+
+  parameter_meta {
+    bam_input: ""
+    whitelist: ""
```

Comment:
Same as above, it would be useful to have brief descriptions here as well. What does this whitelist contain?

Reply:
I guess I should have put "AMBROSE HALP!!" here too :P  I will make sure at least the not so obvious inputs have descriptions with @ambrosejcarr 's help and I'm sure I can fill in the obvious ones like `bam_input`